sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -1319,3 +1332,41 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     }
   }
 }
+
+/**
+ * Converts an AmqpAnnotatedMessage or ServiceBusMessage into a properly formatted
+ * message for sending to the mgmt link for scheduling.
+ *
+ * @internal
+ * @hidden
```

Comment:
I thought we only needed one of these :smile:

Reply:
@HarshaNalluru showed me a spot in our internally generated dev docs where these are still showing up but yes, you're right for production.